client/signup/main.jsx

```diff
@@ -604,9 +604,13 @@ class Signup extends Component {
 	}
 
 	getInteractiveStepsCount() {
-		const flowStepsSlugs = flows.getFlow( this.props.flowName, this.props.isLoggedIn ).steps;
-		const flowSteps = flowStepsSlugs.filter( ( step ) => ! steps[ step ].props?.nonInteractive );
-		return flowSteps.length;
+		// Only show progress bar for these two flows
+		if ( ! [ 'newsletters', 'link-in-bio' ].includes( this.props.flowName ) ) {
+			return null;
+		}
```

Comment:
I think this would remove the `Step x of y` [used](https://github.com/Automattic/wp-calypso/pull/66337/files#diff-1ee63778838c31a9ed28f17a1a7139d6304a844cfba059c5ffcd128b175208d3R769) in other flows where we do not have the progress bar?

Reply:
Oh hoo nice catch! Thnx